### ISSUE TITLE
Complete the class, take help from readme file.

### ISSUE THREAD

**Issue by AbhimanyuSinghh** [AbhimanyuSinghh/inheritance#1]
Hi, so I created a simple program explaining Inheritance. Here are two subclasses of a class Learn. Write the complete functionality of the class methods in each class, where a user can signUp with proper credentials and then he can signIn in order to register for a course. Similarly, you have to write the same sin=gnUp and logIn for the Instructor as well as he is the one who will provide the courses to register, this is where inheritance will help.
And then ofcourse Instructor has to create the course first so write the create function as well and then ask for a review from the student and then instructor will answer to the review or the query by the instructor.

Basically, every static method and the class method has to be written properly so that it can exhibit the behaviour of an online learning site where a student and an instructor can signUp and login and register or create their courses.
